test/state/state.cpp

```diff
@@ -136,11 +136,26 @@ std::variant<int64_t, std::error_code> validate_transaction(const Account& sende
 }
 
 void finalize(State& state, evmc_revision rev, const address& coinbase,
-    std::optional<uint64_t> block_reward, std::span<Withdrawal> withdrawals)
+    std::optional<uint64_t> block_reward, std::span<Withdrawal> withdrawals,
+    std::span<Ommer> ommers)
 {
     // TODO: The block reward can be represented as a withdrawal.
     if (block_reward.has_value())
+    {
         state.touch(coinbase).balance += *block_reward;
+        if (!ommers.empty())
+        {
+            state.touch(coinbase).balance += *block_reward * ommers.size() / 32;
```

Comment:
I think no need to `touch()` because it's always touched above, can be just `get()`.